webapi/src/main/scala/org/knora/webapi/util/ActorUtil.scala

```diff
@@ -263,3 +263,35 @@ trait Task[T] {
     previousResult: Option[TaskResult[T]]
   )(implicit timeout: Timeout, executionContext: ExecutionContext): Future[TaskResult[T]]
 }
+
+trait ResultAndNext[T] {
```

Comment:
Is the `org.knora.webapi.util.Task` and its result still in use, or can you remove this now?

Reply:
Good point, I simply forgot to check it due to drowning into debugging 😅 Done in [ab3bdec](https://github.com/dasch-swiss/dsp-api/pull/2498/commits/ab3bdec92de0f577c2353d7e84e6ba3b9936f415)